miio/wifirepeater.py

```diff
@@ -0,0 +1,97 @@
+import logging
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WifiRepeaterStatus:
+    def __init__(self, data):
+        """
+        Response of a xiaomi.repeater.v2:
+
+        {
+          'sta': {'count': 2, 'access_policy': 0},
+          'mat': [
+            {'mac': 'aa:aa:aa:aa:aa:aa', 'ip': '192.168.1.133', 'last_time': 54371873},
+            {'mac': 'bb:bb:bb:bb:bb:bb', 'ip': '192.168.1.156', 'last_time': 54371496}
+          ],
+          'access_list': {'mac': ''}
+        }
+        """
+        self.data = data
+
+    @property
+    def access_policy(self) -> int:
```

Comment:
Is it known what this integer represents?

Reply:
Unfortunately, no. My assumption is the connectivity between the clients. I cannot be controlled at the moment. The method is unknown / unused of the Mi Home app.